### PR TITLE
Add specs

### PR DESCRIPTION
**Pull request by Phil-Friderici**

### CODE REVIEW

**Comment by Phil-Friderici** [comment]
This patch adds full spec tests. Additionally it fixes the missing path issue for unqualified binaries.

File: manifests/gconf.pp [hunk 1]
```diff
@@ -26,12 +26,14 @@
     command => $command,
     unless  => $unless,
     require => User[$user],
+    path    => '/bin:/usr/bin',
```
Comment (raphink) [inline manifests/gconf.pp]: Please default to `$::path`.

File: manifests/gconf.pp [hunk 2]
```diff
@@ -26,12 +26,14 @@
...
   }
   if $schema {
     exec {"set default schema entry for ${keyname} in ${user}":
       command => "su -l -c 'gconftool-2 --apply-schema ${schema} ${keyname}' ${user}",
       unless  => "test \"${schema}\" == \"$(su -l -c '/usr/bin/gconftool-2 --get-schema-name ${keyname}' ${user})\"",
       require => Exec["set param ${keyname} with value ${value} for user ${user}"],
+      path    => '/bin:/usr/bin',
```
Comment (raphink) [inline manifests/gconf.pp]: Same here, default to `$::path`

**Comment by raphink** [comment]
This looks good, please just use `$::path` instead of a hardcoded value.

**Comment by Phil-Friderici** [comment]
Thanks @raphink for reviewing :)
Good point to use $::path here, have changed the code & tests accordingly.

**Comment by raphink** [comment]
I just updated `master` to fix some tests. Could you rebase your branch on it please?

**Comment by Phil-Friderici** [comment]
et voilà

**Comment by raphink** [comment]
Merci :)

**Pull request merged**